packages/core/strapi/src/index.ts

```diff
@@ -7,9 +7,6 @@ export type * from '@strapi/types';
 
 export type { StrapiCommand, CLIContext } from './commands/types';
 
-export * as factories from './factories';
+// export * as factories from './factories';
```

Comment:
Why has this been removed?

Reply:
I'll remove it for now because I'm going to change the API